orbit_predictor/utils.py

```diff
@@ -297,9 +301,12 @@ def _sun_eci(w, M, L, eccentricity, oblecl):
     zeclip = 0.0
```

Comment:
this line isn't needed anymore, and flake8 is complaining in the pipelines about it

Reply:
I commented it out. I tried to leave the name somewhere so the formula is easier to be recognized.